### ISSUE TITLE
Improve accessibility SfTextInputLayout

### ISSUE THREAD

**Issue by jfversluis** [syncfusion/maui-toolkit#66]
### Description

As reported on the .NET MAUI repository [here](https://github.com/dotnet/maui/issues/26916), the accessibility of the enclosed `Entry`s in a `SfTextInputLayout` is reduced as screen readers do not see the `Entry` control and skip over them.

I have selected Android and macOS right now as affected platforms, but I assume that this is something that is applicable to all platforms.

### Steps to Reproduce

(Taken from the issue reported on the .NET MAUI issue linked above)

1. Turn on Talkback and Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till Hamburger menu and press enter, Projects options will screen will open.
4. Tab till New button and press enter, New project screen will appear, Verify all the elements in Project screen are accessible and MAS Compliant.
5. Choose a category from the list of category.
6. Choose a tag from the list of tags.
7. We can select the task from the existing task list.
8. Observe that In New project screen Name, category and description sections are not accessible with talkback

### Version with bug

1.0.2

### Is this a regression from previous behavior?

No, this is a new issue

### Last Known Working Version

1.0.2

### Affected platforms

Android, macOS

**Comment by jfversluis** [comment]
I just discovered the `AutomationProperties.IsInAccessibleTree` property. That at least allows us to see it with the screen reader without any changes on your side. However, I do still think its not ideal. If I press on it, it reveals the `Entry` but I can't start typing. Also the macOS Accessibility Inspector doesn't then recognize it as a textbox yet.

Just adding this for extra information. Maybe it will help you resolve it :)

```
<sf:SfTextInputLayout 
                    Hint="Name" AutomationProperties.IsInAccessibleTree="True">
    <Entry Text="{Binding Name}" />
</sf:SfTextInputLayout>
```
